### PR TITLE
Place wheels to a separate folder

### PR DESCRIPTION
**Pull request by mryzhov**
### Details:
 - Changed cpack dir name where the wheels should be installed 

### Tickets:
 - *153216*

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by akladiev** [comment]
Please also update here https://github.com/openvinotoolkit/openvino/blob/46736bc010bbceadf37a6511105654dc913139e1/.github/actions/openvino_provider/action.yml#L140

And probably also add a new archive name to defaults here https://github.com/openvinotoolkit/openvino/blob/46736bc010bbceadf37a6511105654dc913139e1/.github/actions/restore_artifacts/restore_artifacts.py#L66, otherwise provider call in tokenizers workflows should be updated to request wheels package explicitly

File: .github/workflows/job_build_linux.yml [hunk 1]
```diff
@@ -178,31 +178,23 @@ jobs:
...
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_package.tar.gz *
+        working-directory: ${{ env.INSTALL_DIR }}
+        
+      - name: Pack openvino_developer_package
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_developer_package.tar.gz *
+        working-directory: ${{ env.DEVELOPER_PACKAGE_DIR }}
+        
+      - name: Pack openvino_tests
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_tests.tar.gz *
+        working-directory: ${{ env.INSTALL_TEST_DIR }}
```
Comment (ilya-lavrenov) [inline .github/workflows/job_build_linux.yml:196]: should we pack wheels as dedicated package? they are typically not a part of OpenVINO release package

Comment (mryzhov) [reply .github/workflows/job_build_linux.yml:196]: yes, we should, I'm working on that now, the last changes did introduce that, but still need to modify pipelines

Comment (mryzhov) [reply .github/workflows/job_build_linux.yml:191]: done

**Comment by ilya-lavrenov** [comment]
We need to merge PRs to GenAI and Tokenizers right after this PR to avoid broken precommits.

**Pull request merged**